scripts/repair_item_sections.py

```
@@ -0,0 +1,592 @@
+                    "items" if cls == "C" else "zone",
+                )
+                delta = round(base0 - (sum0 or 0), 2)
+                tol = max(0.02, base0 * 0.01)
+                combo = subset_closing_gap(cands, delta, tol)
+                if combo is None:
+                    rec["decision"] = "skip: no subset closes the gap"
+                    rec["n_candidates"] = len(cands)
+                    journal.propose(rec)
+                    stats[f"{cls}-no-fit"] += 1
+                    continue
+                added = sorted({l for i in combo for l in cands[i]["lids"]})
```

Ruff (0.16.0)

[error] Ambiguous variable name: `l`

(E741)